erigon-lib/commitment/hex_patricia_hashed.go

```diff
@@ -2049,7 +2049,7 @@ func (hph *HexPatriciaHashed) GenerateWitness(ctx context.Context, updates *Upda
 		//hph.updateCell(plainKey, hashedKey, update)
 
 		// convert grid to trie.Trie
-		tr, err = hph.ToTrie(hashedKey, codeReads) // build witness trie for this key, based on the current state of the grid
+		tr, err = hph.witnessToTrie(hashedKey, codeReads) // build witness trie for this key, based on the current state of the grid
```

Comment:
This reads a bit weird. `ToWitnessTrie` sounds more accurate (although I understand it would break the patter on starting the function name by `witness` ).